editor/create_dialog.cpp

```diff
@@ -425,11 +431,20 @@ void CreateDialog::_sbox_input(const Ref<InputEvent> &p_ie) {
 		switch (k->get_keycode()) {
 			case Key::UP:
 			case Key::DOWN:
+			case Key::LEFT:
+			case Key::RIGHT:
 			case Key::PAGEUP:
 			case Key::PAGEDOWN: {
 				search_options->gui_input(k);
 				search_box->accept_event();
 			} break;
+			case Key::SPACE: {
```

Comment:
Pretty sure this wasn't here when I checked previously .-.

Reply:
On another though, this makes more sense, because node names can't have spaces. But then you'd need to remove left/right arrow handling.

Reply:
But in other trees (SceneTree, FileTree) SPACE does the same as ENTER.

Reply:
Most people likely use Enter though. Also you can't even use Space while typing, you need to click an item to select it first.